### PR TITLE
Add Sophos Central

### PR DESCRIPTION
**Pull request by arnydo**

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]
@arnydo thanks for the pr!

File: _data/security.yml [hunk 1]
```diff
@@ -237,6 +237,14 @@ websites:
       twitter: SaferVPN
       facebook: SaferVPNCom
 
+    - name: Sophos Central
+      url: https://central.sophos.com
```
Comment (Carlgo11) [inline _data/security.yml]: The main url seems to be `https://www.sophos.com`. Please change the value of the `url` tag to that.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/security.yml (hunk 1 above)
Comment (arnydo) [reply _data/security.yml]: I can certainly change to the requested URL, however, they have several products and not all support MFA... Sophos Central is one that Does support MFA.

File: _data/security.yml [hunk 2]
```diff
@@ -237,6 +237,14 @@ websites:
       twitter: SaferVPN
       facebook: SaferVPNCom
 
+    - name: Sophos Central
+      url: https://www.sophos.com
+      img: sophoscentral.png
+      tfa: yes
```
Comment (Carlgo11) [inline _data/security.yml]: Please capitalize all `Yes` values

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by arnydo — COMMENTED** [review]
Updated the "yes" values. Capitalized each.

**Comment by Carlgo11** [comment]
For the email part of the authentication, is that instead of a password?

**Comment by arnydo** [comment]
> 
> 
> For the email part of the authentication, is that instead of a password?

No, it is in addition to the password. The email contains a OTP (4 digit code) as an alternative to Google Authenticator.

**Review by brycx — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by carlylemiii** [comment]
This should have included "Exceptions" like done for Wikipedia and OpenDNS, for example, and been named just Sophos if the URL wasn't going to be specific to Sophos Central. 🙁

**Comment by Carlgo11** [comment]
I’m not sure I understand what you mean @carlylemiii.
The listing is for Sophos Centeal and not any other products.

Our guidlines require the url tag to link to the service’s main page. It also can’t be linked to a login page or similar.

The name should make it obvious that it’s only for Sophos Central and not any other product.

**Pull request merged**